tasks/main.yml

```diff
@@ -256,6 +256,27 @@
       fail:
         msg: "{{ ansible_failed_result.stderr_lines }}"
 
+- name: Ensure the directory and permissions {{ item }}
+  when: item is not none
+  loop:
+    - "{{ mssql_datadir }}"
+    - "{{ mssql_logdir }}"
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: mssql
+    group: mssql
```

Comment:
The mode of the dirs created by default - `/var/opt/mssql/data` and `/var/opt/mssql/logs` - is `0700`. Mode is the only additional parameter here that users might want to configure, owner and group must be `mssql`. So probably it would be easier to add additional variables `mssql_datadir_mode` and `mssql_logdir_mode` that default to 0700 and can be customized.